tests/backend/test_headers.py

```diff
@@ -12,3 +13,18 @@ def test_ip_resolver_not_returned(graphql_endpoint, query_me):
     res = requests.post(graphql_endpoint, json=query_me)
     with pytest.raises(KeyError):
         res.headers["X-Resolver-IP"]
+
+
+def test_security_headers_need_highest_possible_grade(domain):
+    res = requests.get('https://securityheaders.com/?q=' +
+                       domain + '&followRedirects=on')
+    assert res.headers['X-Grade'] == 'A+'
+
+
+def test_ssllabs_scan_grade_is_highest(domain):
+    cmd = sh.Command('../../ssllabs-scan')
+    result = cmd('-grade', '-json-flat', '-verbosity', 'error', domain)
```

Comment:
is it really necessary to have all those args here? isn't 
```
result = cmd('-grade', domain)
```
enough?

Reply:
maybe, need to try it. will check later